WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -972,8 +972,9 @@ extension OrderDetailsDataSource {
             if shouldShowReceipts {
                 rows.append(.seeReceipt)
             }
-
-            if !isRefundedStatus && !isEligibleForCardPresentPayment {
+            let orderTotal = Double(self.order.total) ?? 0
```

Comment:
Great work making this a separate check - this clarifies your intention nicely 👍 
However I think `self` isn't really necessary here, and our overall coding convention (I can't really find the reference anywhere though 😓) that we omit `self` whenever possible.